apps/web-app/emails/invites.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+  import { defineProps, withDefaults } from 'vue';
+  import {
+    EBody,
+    EColumn,
+    EContainer,
+    EHead,
+    EHeading,
+    EHtml,
+    EImg,
+    EPreview,
+    ERow,
+    ESection,
+    EText,
+    ETailwind,
+    ELink,
+    EHr
+  } from 'vue-email';
+
+  interface Props {
+    orgName?: string;
+    inviteCode?: string;
+    expiryDate?: string;
```

Comment:
this should be a `Date` as stated in the original issue :)

Reply:
Thank You : )